lib/downloaderPromise.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+
+const util = require('util')
+const downloader = require('./downloader')
```

Comment:
@brunobg 

brunobg what do you think to just modify `'./downloader'` to use promise-based version if `callback` is not provided?

keep a different file for the same action is confusing.

Reply:
I was trying to avoid mixing them up, but if you have nothing against it I modified the PR.